repo/packages/M/marathon/1/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "images" : {
+    "icon-small" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-small.png",
+    "icon-medium" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-medium.png",
+    "icon-large" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-large.png"
+  },
+  "assets" : {
+    "container" : {
+      "docker" : {
+        "ebbc8372a1c4" : "mesosphere/marathon:v0.15.3"
```

Comment:
We should reference this in `marathon.json.mustache`. Checkout out:

https://github.com/mesosphere/universe/blob/version-2.x/repo/packages/M/marathon/0/marathon.json.mustache#L26-L32